app/assets/stylesheets/base.scss

```diff
@@ -11,26 +11,36 @@ a {
 
   &:focus,
   &:hover {
-    border-bottom: 1px dotted $sul-link-color-border;
+    border-bottom: 1px dotted $sul-link-border-color;
     color: $black;
     text-decoration: none;
   }
 }
 
 body {
+  background-color: $sul-body-bg-color;
+  color: $black;
   font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+  font-size: $font-size-base;
+  line-height: $body-line-height;
+}
+
+h1,
+h2 {
+  font-weight: 300;
 }
 
 h1 {
   color: $sul-h1-font-color;
-  font-weight: 300;
+  font-size: 2.3em;
   letter-spacing: -0.05em;
   padding: 10px 0;
 }
 
 h2 {
   border-bottom: 1px solid $sul-h2-border-color;
-  color: $cardinal-red;
+  color: $sul-h2-font-color;
+  font-size: 1.6em;
   letter-spacing: 0;
   line-height: 1.2em;
   margin: 0.5em 0;
```

Comment:
@jvine is the intention to have the `<h2>` styled globally to have the underline  (i.e. the SearchWorks title and email below)?

It seems that #93 describes that the underline should only actually apply to the home page.

![h2s](https://cloud.githubusercontent.com/assets/96776/7426795/18d3d454-ef83-11e4-8934-86c86cd1289c.png)